### PR TITLE
Add Dropout random seed

### PR DESCRIPTION
**Pull request by keigohtr**
## What is this PR for?

Add Dropout random seed option.

## This PR includes

- Add environment variable for Dropout random seed

## What type of PR is it?

Feature

## What is the issue?

https://github.com/abeja-inc/platform-template-image-classification/issues/2

## How should this be tested?

N/A

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -41,7 +42,7 @@ def create_model(num_classes, input_shape):
     top_model = Sequential()
     top_model.add(Flatten(input_shape=vgg16.output_shape[1:]))
     top_model.add(Dense(128, activation='relu'))
-    top_model.add(Dropout(DROPOUT))
+    top_model.add(Dropout(DROPOUT, seed=DROPOUT_SEED))
```
Comment (ysku) [inline train.py:45]: 関係ないですが、keras のドキュメントの参考文献に論文のリンクあって、おおってなりました
https://keras.io/ja/layers/core/

**Review by ysku — APPROVED** [review]
LGTM 👍

File: train.py (hunk 1 above)
Comment (keigohtr) [reply train.py:45]: そうなんですよね。はるか昔に読んだ気がします。。。
